CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+### 0.8.0
+
+- **Breaking Change** Connection requirement of metadata formulas are now set to optional. If your pack is using sematic versions, this will likely lead to a major version bump in your next release.
```

Comment:
Nit, just for comprehensive, maybe "The connection requirement for metadata formulas now defaults to optional instead of required."